.swiftlint.yml

```diff
@@ -34,7 +35,6 @@ opening_brace: error
 return_arrow_whitespace: error
 statement_position:
   severity: error
-todo: warning
```

Comment:
this should stay to keep track of TODOs.

Reply:
I have removed it temporarily as I will be adding Todos during refactoring to take care of later. I will enable it when we merge to main.